### ISSUE TITLE
Phylogenetic comparative analyses?

### ISSUE THREAD

**Issue by rozdakin** [minisciencegirl/studyGroup#32]
**Where:** [Biodiv 224, UBC](https://www.google.ca/maps/place/Beaty+Biodiversity+Museum,+Vancouver,+BC+V6T+1Z4/@49.2634569,-123.2500362,17z/data=!4m2!3m1!1s0x548672c9f84be123:0x1c431143106eb830)
**When:** 3:00 PM, August 26
**Setup:** [see this comment](https://github.com/minisciencegirl/studyGroup/issues/32#issuecomment-134693562)

How about a workshop on this at some point? Just starting to fumble my way through a comparative analysis. I'd love to learn from someone with more experience.

**Comment by katieannemills** [comment]
Sounds awesome, any takers? @radaniba @ttimbers and other people of the genetics persuasion?

Also, @rozdakin if you've been working with this, I bet you know enough to get us started - why don't we start with a lesson from you on what you've done so far? I suspect there'll be a pile-on effect afterwards, where everyone else interested in phylogenetics will want to jump in with their own stuff once you get the ball rolling.

**Comment by rozdakin** [comment]
Could do, though not until September at the earliest!

To clarify, I'm talking about analyses that look at trait variation/associations across multiple species. i.e., controlling for or applying a phylogeny, not creating one

**Comment by katieannemills** [comment]
Cool! No hurries, whenever is convenient for you is more than perfect. I think topic-focused series of events are a great idea, we should all keep eyes out for possible speakers; I certainly will!

**Comment by radaniba** [comment]
@BillMills  @rozdakin  Sounds great but unfortunately beyond my scope even though I did a lot of alignment/phylogeny before, but it was while ago. I would attend though, it sounds interesting

**Comment by radaniba** [comment]
I found this recently it is relevant to this issues.

@rozdakin you might find this useful

http://www.bioconductor.org/packages/3.1/bioc/vignettes/ggtree/inst/doc/ggtree.html

**Comment by minisciencegirl** [comment]
Hi Roz, I found this link to be super useful for my bacterial genomics and may be helpful: https://bacpathgenomics.wordpress.com/2012/05/25/displaying-data-associated-with-phylogenetic-trees/

**Comment by mwpennell** [comment]
My name is Matt Pennell and I just arrived at UBC. I would be happy to do a tutorial on phylogenetic comparative analyses/working with and plotting trees in R.

**Comment by radaniba** [comment]
I tried the python ete2 package for fun before and it is easy to use http://coderscrowd.com/app/public/codes/view/141

**Comment by minisciencegirl** [comment]
That sounds awesome!!! What date would work for you?

Amy

**Comment by rozdakin** [comment]
Thanks @radaniba! After some sleuthing, I think I've figured out that the specific analysis I'd like to do is not possible in an existing R package.
Matt, I'd be keen on this. How long are you at UBC for?

**Comment by minisciencegirl** [comment]
Hi @mwpennell! Just confirming the dates for the two workshops you were talking about - 8/19 & 8/26 at 3pm still work for you? @rozdakin: do these times work for you as well? Cheers!

**Comment by rozdakin** [comment]
yes to 8/26, but no to 8/19 unfortunately

**Comment by mwpennell** [comment]
Yep. These work for me. Looking forward to it

**Comment by minisciencegirl** [comment]
@mwpennell Could we move one lesson to the following week? 8/26 and 9/2 at 3pm?

**Comment by minisciencegirl** [comment]
Hi @rozdakin: We moved the lessons to 8/26 and 9/2 at 3PM for Part I and II. @mwpennell: do you mind booking the BioDiv 224 room for us? Thank you!!

**Comment by rozdakin** [comment]
@mwpennell Are we still doing this on Wed 8/26? I see room 224 isn't booked. If we're still on, I can book it. Let me know!

**Comment by mwpennell** [comment]
Yep, I'm still on for doing this. Sorry for not booking the room; I completely forgot :(

I'm not on campus today. If you could book it, that would be great! Thanks so much @rozdakin

**Comment by rozdakin** [comment]
Alright! Room 224 Biodiv is booked for R phylo stuff on Wed 8/26 at 3PM.
Looking forward to it.

**Comment by minisciencegirl** [comment]
Thanks @rozdakin for booking the room! Would you mind also booking the same room for the following week, 9/2, 3pm, for Part 2?

@mwpennell: Would you be open to live webcasting for this tutorial? We tried it last week for the biomaRt tutorial and it went really well. I unfortunately won't be able to make it for Part 1 because something came up. @BillMills can help you set up the live webcasting. Thanks!!

**Comment by mwpennell** [comment]
@minisciencegirl I would prefer not to do live webcasting. While I appreciate the potential value, having a lifestream would make me exponentially more nervous about presenting. I realize that I'm being irrational but it is what it is. Hopefully that is ok.

**Comment by minisciencegirl** [comment]
Not a problem @mwpennell! This is why we check in with our presenters first. :-) When you get a chance, if you could please let us know the packages we should install ahead of time and/ or companion scripts, that will be great. Thanks again!

**Comment by rozdakin** [comment]
Already done!

**Comment by mwpennell** [comment]
I am (slowly, but surely) adding materials (scripts and datasets) to the a gh repo https://github.com/mwpennell/r-phylo

The packages that we will be using tomorrow will be
- ape
- geiger
- nlme
- diversitree (possibly; we'll see how far we get)

I have tried to set up my repo such that all vignettes can be automatically rebuilt using the [remake](https://github.com/richfitz/remake) R package. Hopefully this works on your end. 

I will try to add a bit more later today. See you all tomorrow!

**Comment by rozdakin** [comment]
Thanks Matt!

By the way, for background on comparative analysis I like <a href="http://link.springer.com/chapter/10.1007/978-3-662-43550-2_5"> this primer</a>.

**Comment by rozdakin** [comment]
Update: we're putting Part II on hold for now, cancelling the workshop on Sept 2.

For anyone interested, I'm now working with R's MCMCglmm to look at inter- AND intraspecific variation in relation to multiple predictors, accounting for phylogeny. (Thanks, @mwpennell!) Happy to chat if others are interested.

**Comment by minisciencegirl** [comment]
Thanks Roz. Why don't we have a work-along instead?

**Comment by rozdakin** [comment]
I'd be down. The room booking stands!
